app/grandchallenge/ai_website/models.py

```diff
@@ -0,0 +1,170 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.text import get_valid_filename
+
+from grandchallenge.core.storage import public_s3_storage
+
+
+def get_logo_path(instance, filename):
+    return f"logos/{instance.__class__.__name__.lower()}/{instance.pk}/{get_valid_filename(filename)}"
+
+
+def get_images_path(instance, filename):
+    return f"product_images/{instance.__class__.__name__.lower()}/{instance.pk}/{get_valid_filename(filename)}"
+
+
+class CompanyEntry(models.Model):
+    # author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    created_date = models.DateField(default=timezone.now)
+    modified_date = models.DateField(auto_now=True)
+    company_name = models.CharField(max_length=200)
+    website = models.URLField(max_length=500)
+    founded = models.IntegerField()
+    hq = models.CharField(max_length=100)
+    email = models.EmailField(max_length=500)
+    logo = models.ImageField(
+        upload_to=get_logo_path, storage=public_s3_storage, null=True,
+    )
+    description = models.CharField(
+        max_length=500,
+        default="",
+        blank=True,
+        help_text="Short summary of this project, max 500 characters.",
+    )
+    description_short = models.CharField(
+        max_length=250,
+        blank=True,
+        help_text="Short summary of this project, max 250 characters.",
+    )
+
+    def __str__(self):
+        return self.company_name
+
+
+class ProductImage(models.Model):
+    img = models.ImageField(
+        upload_to=get_images_path, storage=public_s3_storage
+    )
+
+
+# Create your models here.
+class ProductEntry(models.Model):
+    STATUS_CERTIFIED = "cer"
+    STATUS_YES = "yes"
+    STATUS_NO = "no"
+    STATUS_NA = "na"
+    STATUS_CLEARED = "cle"
+    STATUS_DE_NOVO_CLEARED = "dnc"
+    STATUS_PMA_APPROVED = "pma"
+    STATUS_UNKNOWN = "unk"
+
+    VERFIFIED_CHOICES = ((STATUS_YES, "Yes"), (STATUS_NO, "No"))
```

Comment:
We can use Enumeration types now :-) https://docs.djangoproject.com/en/3.0/ref/models/fields/#field-choices-enum-types